setup.py

```diff
@@ -53,8 +53,12 @@ class BuildProtoCommand(Command):
   def initialize_options(self):
     self.skip_proto = False
     try:
-      prefix = subprocess.getoutput(
-          'pkg-config --variable prefix protobuf').strip()
+      if sys.version_info[0] < 3:
```

Comment:
Does http://python-future.org/compatible_idioms.html#subprocess-check-output not let us avoid the test for 2 vs 3?